language/move-prover/src/prelude.bpl

```diff
@@ -1033,6 +1033,9 @@ axiom (forall ta1, ta2: TypeValue ::
        (forall v1, v2: Value :: IsEqual($serialize(ta1, v1), $serialize(ta2, v2))
            ==> IsEqual(v1, v2) && (ta1 == ta2)));
 
+function $LCS_serialize(ta: TypeValue, v: Value): Value {
```

Comment:
It looks like we could rename "$serialize" everywhere to "$LCS_serialize" and save this redundant function definition.

Reply:
Done renaming.